statsd/options.go

```diff
@@ -23,16 +34,20 @@ type Options struct {
 	Namespace string
 	// Tags are global tags to be applied to every metrics, events and service checks.
 	Tags []string
-	// Buffered allows to pack multiple DogStatsD messages in one payload. Messages will be buffered
-	// until the total size of the payload exceeds MaxMessagesPerPayload metrics, events and/or service
-	// checks or after 100ms since the payload startedto be built.
-	Buffered bool
+	// MaxMessagesPerPayload is the maximum number of bytes a single payload will contain.
+	// The magic value 0 will set the option to the optimal size for the transport
+	// protocol used when creating the client: 1432 for UDP and 8192 for UDS.
+	// Note that this option only takes effect when the client is buffered.
+	MaxBytePerPayload int
 	// MaxMessagesPerPayload is the maximum number of metrics, events and/or service checks a single payload will contain.
 	// Note that this option only takes effect when the client is buffered.
 	MaxMessagesPerPayload int
```

Comment:
Since the client is buffered by default, I believe the only way for the client not to buffer the messages is to set this option to 1, so the documentation should probably be amended to reflect this.